### PR TITLE
feat: adds talks to summit

### PR DESCRIPTION
**Pull request by prosazhin**
This pull request adds the talks to summit 2023 pages.
[summit-2023](https://deploy-preview-502--ebpfio.netlify.app/summit-2023/)
[summit-2023-schedule](https://deploy-preview-502--ebpfio.netlify.app/summit-2023-schedule/)
[summit-2023-talks](https://deploy-preview-502--ebpfio.netlify.app/summit-2023-talks/)

### CODE REVIEW

**Comment by prosazhin** [comment]
@xmulligan Hello. Please take a look at what we've accomplished. There's still a question: should we rename the 'Schedule' page to 'Talks' in the navigation and the page URL?

**Comment by prosazhin** [comment]
@xmulligan Hello. Please take a look at what we've accomplished.

**Comment by xmulligan** [comment]
That looks good so far and thanks for updating to talks. However it seems that the links to all of the talks are missing?

**Comment by xmulligan** [comment]
on the homepage, the button should also be updated to "Watch the recordings on YouTube" and similarly with the banner

**Comment by prosazhin** [comment]
@xmulligan Hello. I apologize. It seems I didn't fully understand what needs to be changed. Could you please provide a step-by-step list of the remaining text and links that need to be modified to complete this task?

**Comment by xmulligan** [comment]
Screenshot 2023-10-24 at 15 36 31
Screenshot 2023-10-24 at 15 36 36
These two should be updated. I've also add links to all of the talks in sessionize

**Comment by prosazhin** [comment]
@xmulligan I fixed the button on the main page and the banner. Also, I enabled sending 'recordingUrl' in Sessionize settings. Now, the video watch button is accessible in the list of talks.

**Comment by xmulligan** [comment]
one last things, can you update the register part to "Find all of the recordings on YouTube" and the button "Watch now"
Screenshot 2023-10-24 at 18 13 04

**Comment by prosazhin** [comment]
@xmulligan I updated. But what should "Watch now" do now?

File: src/components/pages/summit-2023/hero/hero.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const ctaButtons = [
   {
     onClick: null,
     url: 'https://www.youtube.com/watch?v=EViAho-6qoc',
```
Comment (xmulligan) [inline src/components/pages/summit-2023/hero/hero.jsx:17]:
```suggestion
    url: 'https://www.youtube.com/playlist?list=PLDg_GiBbAx-l4D4oKbscJhPFKv2oqPcD_',
```

File: src/components/shared/summit-top-banner/summit-top-banner.jsx [hunk 2]
```diff
@@ -17,11 +17,11 @@ const SummitTopBanner = () => (
         aria-hidden
       />
       <span className="font-sans text-lg font-medium leading-none text-white md:text-base">
-        YouTube stream of eBPF Summit
+        Watch the recordings on YouTube
       </span>
     </div>
     <Link theme="orange" size="lg" to="https://www.youtube.com/watch?v=EViAho-6qoc" target="_blank">
```
Comment (xmulligan) [inline src/components/shared/summit-top-banner/summit-top-banner.jsx:23]:
```suggestion
    <Link theme="orange" size="lg" to="https://www.youtube.com/playlist?list=PLDg_GiBbAx-l4D4oKbscJhPFKv2oqPcD_" target="_blank">
```

**Comment by xmulligan** [comment]
@prosazhin it should go to https://www.youtube.com/playlist?list=PLDg_GiBbAx-l4D4oKbscJhPFKv2oqPcD_

**Comment by prosazhin** [comment]
@xmulligan Ready. You can check it

**Comment by xmulligan** [comment]
Thank you @prosazhin !

**Pull request merged**